codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/DirectedRubyCodegen.java

```diff
@@ -182,6 +183,9 @@ public void generateService(GenerateServiceDirective<GenerationContext, RubySett
         GlobalConfigPluginGenerator globalConfigPluginGenerator = new GlobalConfigPluginGenerator(directive);
         globalConfigPluginGenerator.render();
 
+        CustomizationsGenerator customizationsGenerator = new CustomizationsGenerator(directive);
+        customizationsGenerator.render();
```

Comment:
As an alternative to always writing out a blank customization.rb what about having the servce.rb check for the existence of customization.rb before requiring it, eg:
```ruby
# lib/white_label.rb
# regular requires
require_relative 'white_label/validators'
require_relative 'white_label/waiters'
require_relative 'white_label/event_stream'
if file.exists?(File.join(__dir__, 'white_label/customizations.rb'))
  require_relative 'white_label/customizations'
end

```

Reply:
Good idea, I do a rescue here if the file exists or not.